docs/license.rst

```diff
@@ -1,15 +1,4 @@
 License
 =======
 
-Copywrite 2015 Lifealike Ltd. DBA onefinestay
-
-Nameko is licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
```

Comment:
maybe something like "Nameko is licensed under the Apache License, Version 2.0". Please see `LICENSE` in the project root for details